cloud-sql/mysql/mysql/views/index.pug

```diff
@@ -53,9 +53,7 @@ html(lang="en")
         xhr.onreadystatechange = function () {
           var msg = "";
           if (this.readyState == 4) {
-            if (!window.alert(this.responseText)) {
-              window.location.reload();
-            }
+            window.location.reload();
```

Comment:
this will allow the app to reload seamlessly without an annoying alert popup on screen. This alert is not used in other languages, updating for consistency.